### PR TITLE
feat: add new categories

### PR DESCRIPTION
**Pull request by dtroelofsprins**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/system tested
- [ ] User documentation updated
- [ ] Clean commits
- [x] Updated CHANGELOG.md

### CODE REVIEW

**Comment by dtroelofsprins** [comment]
[#10748 Add derivation of new categories to the publish script](https://trello.com/c/hMCaEYMP/1695-10748-add-derivation-of-new-categories-to-the-publish-script)

File: src/molgenis/bbmri_eric/categories.py [hunk 1]
```diff
@@ -52,6 +78,68 @@ class AgeUnit(Enum):
     "urn:miriam:icd:U07.2",
```
Comment (esthervanenckevort) [inline src/molgenis/bbmri_eric/categories.py:78]: This is outside the change, but I noticed that this isn't the complete list of COVID-19 related terms. U07.1, U08, U09. U09.9, U10, U10.9, U11, U11.9, U12, U12.9 are also COVID19 related

**Review by esthervanenckevort — CHANGES_REQUESTED** [review]

File: src/molgenis/bbmri_eric/categories.py (hunk 1 above)
Comment (dtroelofsprins) [reply src/molgenis/bbmri_eric/categories.py]: Most of them were already there (I think you've overlooked them as the file was not fully shown), but I've added U09.9, U10.9, U11.9 and U12.9

**Review by esthervanenckevort — APPROVED** [review]
Approved when the K5190 code has been resolved

**Pull request merged**